### PR TITLE
Disable all calls for migration

### PR DESCRIPTION
**Pull request by echevrier**
Disable all calls for migration
Bump version: spec version = 9

### CODE REVIEW

**Comment by echevrier** [comment]
Runtime upgrade tested. After the runtime upgrade, transfer are impossible.

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -220,10 +222,31 @@ parameter_types! {
...
+			Call::Balances(..) |
+			Call::Claims(..) |
+			Call::Multisig(_) |
+			Call::Proxy(_) |
+			Call::Teeracle(_) |
+			Call::Teerex(_) |
+			Call::Treasury(..) |
+			Call::Scheduler(_) |
+			Call::Utility(_) |
+			Call::Vesting(_)
```
Comment (brenzi) [inline runtime/src/lib.rs]: what about `system`? If I'm not mistaken, `system.remark()` is not privileged. We may want to filter that too

**Review by brenzi — CHANGES_REQUESTED** [review]
If you do test again, please not only test that balance transfers are filtered, also test that sudo calls are still possible. In the best case, perform another setCode() because that's what we may need to do if things go sideways with the migration. 
(I have no doubt it will work, but let's be very thorough)

File: runtime/src/lib.rs (hunk 1 above)
Comment (echevrier) [reply runtime/src/lib.rs]: Only system call remark? If we want to make the upgrade we need the system setCode.

Comment (brenzi) [reply runtime/src/lib.rs]: sudo can do all calls anyway no matter the filtering

Comment (brenzi) [reply runtime/src/lib.rs]: ...as long as we don't filter sudo itself ;-)

Comment (brenzi) [reply runtime/src/lib.rs]: actually, last time we didn't filter system, so let's do it the same way now

Comment (brenzi) [reply runtime/src/lib.rs]: https://github.com/integritee-network/integritee-node/blob/6a445c05618016cea7057fe3dbf566027f596c0a/runtime/src/lib.rs#L221-L239

**Review by brenzi — APPROVED** [review]

**Review by brenzi — CHANGES_REQUESTED** [review]
one more thing: if you bump spec version, you have to bump pallet versions too. for runtime and node

**Comment by echevrier** [comment]
> If you do test again, please not only test that balance transfers are filtered, also test that sudo calls are still possible. In the best case, perform another setCode() because that's what we may need to do if things go sideways with the migration. (I have no doubt it will work, but let's be very thorough)

Sudo calls are working

**Comment by brenzi** [comment]
There was a misunderstanding about my comment. But it's ok like this too I think. But we need to test again if sudo setCode works to be sure

**Review by haerdib — APPROVED** [review]
LGTM. I was able to upgrade and downgrade the runtime again with two --dev nodes running locally.

**Comment by echevrier** [comment]
> There was a misunderstanding about my comment. But it's ok like this too I think. But we need to test again if sudo setCode works to be sure

I did all tests again

**Review by brenzi — APPROVED** [review]

**Pull request merged**